src/scss/_variables.scss

```diff
@@ -1,7 +1,3 @@
-// Include Bootstrap vars.
-@import "node_modules/bootstrap/scss/functions";
```

Comment:
I was about to merge this, but I think we're getting some errors if we don't import these here because the overrides wouldn't work correctly.

Reply:
Ok, I'll remove this change from this PR and add a separate one for further investigation.